### PR TITLE
Gibber stuff (and fixing a compile warning *glares at Aurora*)

### PR DESCRIPTION
**Pull request by tigercat2000**
Monkeys can no longer be gibbed, because the feed-in-top animation was not ever designed with them in mind. 
Buncha styling changes.

I eventually want to make more modes for the gibber, as well as a nanoUI interface, but it'll probably never happen *cry*

### CODE REVIEW

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -342,7 +342,7 @@
 /datum/game_mode/proc/update_traitor_icons_added(datum/mind/traitor_mind)
 	var/datum/atom_hud/antag/tatorhud = huds[ANTAG_HUD_SOLO]
 	//var/ref = "\ref[traitor_mind]"
- 	tatorhud.join_solo_hud(traitor_mind.current)
+	tatorhud.join_solo_hud(traitor_mind.current)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/traitor/traitor.dm]: I already fixed this is my implants PR =p

**Comment by Fox-McCloud** [comment]
I will miss gibbing monkies and scaring the shit out of people as chef.

**Comment by Fox-McCloud** [comment]
Changelog

File: code/game/gamemodes/traitor/traitor.dm (hunk 1 above)
Comment (Aurorablade) [inline code/game/gamemodes/traitor/traitor.dm]: don't you glare at me..

**Comment by KasparoVy** [comment]
rest in primate :'(

**Pull request merged**